config/babel.prod.js

```diff
@@ -9,8 +9,7 @@
 
 module.exports = {
   presets: [
-    'babel-preset-es2015',
```

Comment:
Huh, it's weird that this isn't es2015-loose. I can't think of any good reason to not use es2015-loose (and correspondingly es2015-loose-native-modules instead of either es2015-native-modules or es2015-webpack) for production.